dewy/documents/models.py

```diff
@@ -5,9 +5,11 @@
 
 
 class CreateRequest(BaseModel):
-    """The name of the collection the document should be added to."""
+    """The name of the collection the document should be added to. Either `collection` or `collection_id` must be provided"""
+    collection: Optional[str] = None
 
-    collection_id: int
+    """The id of the collection the document should be added to. Either `collection` or `collection_id` must be provided"""
+    collection_id: Optional[int] = None
```

Comment:
Would it be more idiomatic to do `collection: int | str`? Eg., you need to provide the name or the number? I guess the risk is that we may not be able to tell if you gave us a name with only numeric digits?